egs2/ogi_kids_speech/asr1/README.md

```diff
@@ -0,0 +1,27 @@
+# RESULTS
```

Comment:
Can you add a comment that this is the short-sentence version?
Please explain the details of which portion in the directory you used, how many recordings, hours, etc.
You can also add TODO to deal with the long recording.